compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Dialog.skiko.kt

```diff
@@ -126,12 +128,18 @@ actual fun Dialog(
     properties: DialogProperties,
     content: @Composable () -> Unit
 ) {
+    val blendMode = if (LocalWindowInfo.current.isWindowTransparent) {
```

Comment:
I find it unusual to use different blending techniques for opaque and transparent windows when the only difference lies in the transparent regions. The background color of dialogs in opaque areas should remain consistent, regardless of the isWindowTransparent attribute.

If our goal is to avoid displaying a gray color over the transparent parts of the window and avoid the `Before` state, we should always draw White color by default (except if `transparent = false`).

Furthermore, we should use a white background for cross-platform consistency, as macOS currently uses gray, while Windows, Linux, Android, and iOS uses white. I can't think of a scenario where we need a system background under a Compose content.

Reply:
Decided that it can be configured on the desktop Window level, where the transparency lives.